src/styles/colors.css

```
@@ -5,6 +5,7 @@
   --link-color-active: #0b6666;
 
   --text-default: #333333;
+  --text-secundary: #595959;
```

eu acho que é **secondary**
```suggestion
  --text-secondary: #595959;
```

---

Boa